pkg/controllers/management/clusterdeploy/clusterdeploy.go

```diff
@@ -112,6 +112,11 @@ func (cd *clusterDeploy) doSync(cluster *v3.Cluster) error {
 		return nil
 	}
 
+	if !clouddrivers.HasPublicAPIEndpoint(cluster) {
+		logrus.Tracef("clusterDeploy: doSync: cluster [%s] is private so agent cannot be deployed automatically", cluster.Name)
```

Comment:
Probably info or debug here, not trace.

Reply:
All of the other logs in this method are Trace. Would you still suggest info or debug?

Reply:
I think debug is fine

Reply:
Done.